R/measures.R

```diff
@@ -1429,10 +1429,8 @@ ibrier = makeMeasure(id = "ibrier", minimize = TRUE, best = 0, worst = 1,
 
     probs = predictSurvProb(getLearnerModel(model, more.unwrap = TRUE), newdata = newdata, times = grid)
     perror = pec(probs, f, data = newdata[, tn], times = grid, exact = FALSE, exactness = 99L,
-      maxtime = max.time, verbose = FALSE)
+      maxtime = max.time, verbose = FALSE, reference = F)
 
-    # FIXME: what is the difference between reference and matrix?
-    # FIXME: this might be the wrong number!
```

Comment:
What about the FIXMEs you deleted? I guess that first one is irrelevant now, but what about the second?

Reply:
Sorry, I thought the second FIXME refers to a possible wrong number caused by the first FIXME. If that's not the case than I'm not sure what is meaning of the second FIXME.  I'll put it back